tools/tcat_ble_client/ble/ble_connection_constants.py

```diff
@@ -0,0 +1,20 @@
+"""
+   Copyright (c) 2023 Nordic Semiconductor ASA
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
```

Comment:
Thanks for adding this.

I noticed the `tcat` files are added in `tools/` folder and they currently have their own license/copyright statements.

To align with OpenThread's licensing policy, we have two options:

- Are you okay to update the licenses in to match the OpenThread license? This would allow us to keep them in the `tools/` folder (which seems reasonable to me).
- Otherwise, we'll need to move all tcat files to the `third_party` folder. This folder is specifically for files with separate licensing terms.

Reply:
@abtink I have updated the License to match the Openthread one.